azkaban-common/src/main/java/azkaban/utils/Utils.java

```diff
@@ -409,4 +408,27 @@ public static String createPeriodString(ReadablePeriod period) {
 
     return periodStr;
   }
+
+  public static long parseMemString2KB(String strMemSize) {
+    if (strMemSize == null) {
+      return 0L;
+    }
+
+    long size = 0L;
+    if (strMemSize.endsWith("g") || strMemSize.endsWith("G")
```

Comment:
It's not necessary that we will have an alphabet in the end for xmx. Things like -Xmx83886080 is also valid in bytes, which will make your code return 0 KB size.